### DISCUSSION TITLE
Channel Bugs

### DISCUSSION THREAD

**Discussion by ahmednaalik51** [BearWare/TeamTalk5#1478] [General]
Hi,
We have tried to report problems with channels to devs but they don't listen.
There is a bug in TeamTalk now that if a person is moved into a channel that has password, or moved at all, then if the connnection will become drop, then it comes back and says that channel password incorect.
This also happens if you set inisial channel with password in connect dialog sometimes it happens as well.
If you want to reproduce, put a admin and a user together, and move the user to the channel that has a password. Don't join the channel, just move them.
Turn off WiFi or whatever, and turn back on after connection drop. You will get error that says incorrect password.
Remember it only happens if they move to the channel, not join, just use the move command.
Also it is only happening in QT client.
Let me know if I needed to provide more information.
Did anyone else got this problems?
Thanks.

**Comment by computertech1012** [comment]
Hi,
I noticed this bug as well, but I also noticed a bug with auto operator channels.
In the latest version, the passwords of auto operator channels are forwarded to clients, but you still are unable to change any settings for the channels until you join the channel. If you do it will give you errors.
Can this also be fixed? It's as if it's not giving the auto operator to the user until they actually join the channel, which is not good, it should give all the operator rights to the channels when it joins the server. This way, if for example someone wants to manage the ban list of their auto operator channel, they can do it without joining.
Thanks for reading!
